File: foundation/foundation 1 month/lesson10/ex_6.c
#include <stdio.h>
int main()
{
	int sum=0;
	for(int i=1; i<=19; i++)
{
	printf("%d |",i);
	sum= sum+i;
}
printf("Sonlarning yiginidisi:%d", sum);
return 0;
}

File: full stack/full stack 1 month/lesson3/homework/ex_6.js
function rev(a)
{
    let sum
    if (a<=0)
    {
        return 0
    }
    if (a%2==0)
    {
        return a+ rev(a-2)
    }
   else
   {
        return rev(a-1)
   }
}
console.log(rev(13))